### DISCUSSION TITLE
unable to resolve class MapReduceIndexManagement

### DISCUSSION THREAD

**Discussion by longyan-github** [JanusGraph/janusgraph#3647] [General]
i want to use mapreduce to reindex ,but when i used mr =new MapReduceIndexManagement(graph),i get the error as fllow:
unable to resolve class MapReduceIndexManagement
how to solve the problem?
can anyone help me

**Comment by hadoopmarc** [comment]
The gremlin repl has lots of classes pre-imported, but apparently not this one. You can look up classes in:
https://javadoc.io/doc/org.janusgraph/janusgraph-hadoop/0.6.3/index.html

The solution for this particular case:
```
gremlin> org.janusgraph.hadoop.MapReduceIndexManagement
==>class org.janusgraph.hadoop.MapReduceIndexManagement
gremlin> import org.janusgraph.hadoop.MapReduceIndexManagement
==>org.apache.tinkerpop.gremlin.structure.*, ...
```
